app/controllers/greetings_controller.rb

```diff
@@ -0,0 +1,6 @@
+class GreetingsController < ApplicationController
+  def random_greeting
+    greetings = ['Hello', 'Hi', 'Hola', 'Bonjour', 'Guten Tag']
+    render json: { greeting: greetings.sample }
+  end
```

Comment:
- [ ] Here you're not fetching the greetings from the database but using the local array. Please make sure to fetch the greetings from the database and return them as JSON.
  - You can consider using this code: ` @greeting = Greeting.order('RANDOM()').first`